client/components/feature-example/test/index.js

```diff
@@ -11,14 +11,13 @@ import { shallow } from 'enzyme';
 import FeatureExample from '../index';
 
 describe( 'Feature Example', function() {
-	it( 'should have Feature-example class', function() {
+	it( 'should have Feature-example class', () => {
 		const featureExample = shallow( <FeatureExample /> );
-		assert.equal( 1, featureExample.find( '.feature-example' ).length );
```

Comment:
Yeah, I get it now – the grammar would be wrong if the length was the first argument.